cinema/views.py

```diff
@@ -34,25 +47,102 @@ class MovieViewSet(viewsets.ModelViewSet):
     queryset = Movie.objects.all()
     serializer_class = MovieSerializer
 
-    def get_serializer_class(self):
+    @staticmethod
+    def arg_to_int(arg) -> list[int]:
+        return [int(str_id) for str_id in arg.split(",")]
```

Comment:
You could write this code using map, e.g.:
```suggestion
        return list(map(int, arg.split(",")))
```
Besides that, sometimes it is better to perform such operations in a `try except` block because users sometimes may pass values that cannot be converted to integers.